public_records_portal/templates/email_templates/email_acknowledgement.html

```diff
@@ -16,6 +16,122 @@
                                         {{ notification_content['additional_information'] }}
                                     {% endif %}
                                 </p>
+                            {#                            ------------------Contact and Request Information----------------#}
+                                <p>
+                                    Request Title: {{ notification_content['contact_info']['request_title'] }}
+                                    <br/>
+                                    Request Description: {{ notification_content['contact_info']['request_description'] }}
+                                    <br/>
+                                    <br/>
+                                    Contact Information:
```

Comment:
Why are we including the requester's contact information in the acknowledgement to the requester?